src/lib/homestore_backend/gc_manager.cpp

```diff
@@ -227,14 +223,23 @@ void GCManager::scan_chunks_for_gc() {
 
 /* pdev_gc_actor */
 
-GCManager::pdev_gc_actor::pdev_gc_actor(uint32_t pdev_id, std::shared_ptr< HeapChunkSelector > chunk_selector,
-                                        std::shared_ptr< GCBlobIndexTable > index_table, HSHomeObject* homeobject) :
-        m_pdev_id{pdev_id},
+GCManager::pdev_gc_actor::pdev_gc_actor(const homestore::superblk< GCManager::gc_actor_superblk >& gc_actor_sb,
+                                        std::shared_ptr< HeapChunkSelector > chunk_selector, HSHomeObject* homeobject) :
+        m_pdev_id{gc_actor_sb->pdev_id},
         m_chunk_selector{chunk_selector},
         m_reserved_chunk_queue{HS_BACKEND_DYNAMIC_CONFIG(reserved_chunk_num_per_pdev)},
-        m_index_table{index_table},
-        m_hs_home_object{homeobject} {
-    RELEASE_ASSERT(index_table, "index_table for a gc_actor should not be nullptr!!!");
+        m_index_table{homeobject->get_gc_index_table(boost::uuids::to_string(gc_actor_sb->index_table_uuid))},
+        m_hs_home_object{homeobject},
+        metrics_{*this} {
+
+    RELEASE_ASSERT(m_index_table, "index_table for a gc_actor should not be nullptr!!!");
+
+    durable_entities_.success_gc_task_count = gc_actor_sb->success_gc_task_count;
+    durable_entities_.success_egc_task_count = gc_actor_sb->success_egc_task_count;
+    durable_entities_.failed_gc_task_count = gc_actor_sb->failed_gc_task_count;
+    durable_entities_.failed_egc_task_count = gc_actor_sb->failed_egc_task_count;
+    durable_entities_.total_reclaimed_blk_count_by_gc = gc_actor_sb->total_reclaimed_blk_count_by_gc;
+    durable_entities_.total_reclaimed_blk_count_by_egc = gc_actor_sb->total_reclaimed_blk_count_by_egc;
```

Comment:
suggest using bytes instead of block

Reply:
https://github.com/eBay/HomeObject/pull/333/files#diff-8b489899017e23bcd85c82d6a3f5f5ca7883cdb21c65f91d499d29f1b895e362:~:text=failed_egc_task_count.load(std%3A%3Amemory_order_relaxed))%3B-,GAUGE_UPDATE(,%7D,-private%3A

when collecting metrics, the bytes will be returned